apps/api/resources/api/config.ttl

```diff
@@ -1,10 +1,25 @@
+PREFIX view-builder: <https://view-builder.described.at/>
+PREFIX dcterms: <http://purl.org/dc/terms/>
 PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
 PREFIX code: <https://code.described.at/>
 PREFIX schema: <http://schema.org/>
 PREFIX knossos: <https://hypermedia.app/knossos#>
 
 <>
   a knossos:Configuration ;
+  # TODO: make these env specific
+  view-builder:publisher <https://ld.stadt-zuerich.ch/statistics/meta/org/4> ;
+  view-builder:publishedViewTemplate
+    [
+      a hydra:IriTemplate ;
+      hydra:template "https://ld.stadt-zuerich.ch/statistics/view/{id}" ;
+      hydra:mapping
+        [
+          hydra:variable "id" ;
+          hydra:property schema:alternateName ;
+          hydra:required true ;
+        ] ;
+    ] ;
```

Comment:
These may be some of the more interesting changes here. Rather than adding more env variables, the configuration is added to the database and accessed at runtime

1. `view-builder:publisher` is used to identify only views published by SSZ in Lindas. otherwise we'd potentially load views from other publishers
2. the `view-builder:publishedViewTemplate` is how the `schema:sameAs` is generated from the metadata object. see
[apps/api/lib/view/create.js](https://github.com/zazuko/ssz-view-builder/pull/46/files#diff-ec1f76d3c92734d19e7b1b2f2cb4074560160021f27e8e26ef733e0ce9926a84)

Reply:
Same base code used to create new identifiers when POSTing to a collection